doc/installation.rst

```diff
@@ -38,75 +48,59 @@ You might need to install the ``curl`` program first, if not already installed.
 
   apt-get install curl
 
-3. Install the pre-built binary packages and its dependencies by running the following commands
+3. Install the pre-built binary packages and their dependencies by running the following commands
 
 ::
 
   apt-get update
   apt-get install eos
 
-4. On older version of Ubuntu, you might need to update your installation of the Matplotlib Python package.
-   We recommend to install the most up-to-date Python package with from the Python Package Index, using
+4. On older versions of Ubuntu, you might need to update your installation of the Matplotlib Python package.
+   We recommend installing the most up-to-date Python package from the Python Package Index, using
 
 ::
 
   pip3 install --user h5py matplotlib numpy pypmc pyyaml scipy
```

Comment:
Should we use ``--update`` here? Does it fail, if the packages are not yet installed?

Reply:
no, if the package is already installed, it either updates it or sends a message like "latest version already installed".

Reply:
For what we describe, the first installation of eos, using no flags does the job and has no "side effects".

I propose to add the explicit comment instead that EOS can be updated with the same command except for the additional flag `--upgrade` or `-U` for short.

Reply:
I agree!

Reply:
Except that the flag is ``--upgrade``